src/ui/public/chrome/config/filter.html

```diff
@@ -1,6 +1,6 @@
 <kbn-timepicker
   time="timefilter.time"
-  mode="timefilter.time.mode"
   active-tab="'filter'"
-  interval="timefilter.refreshInterval">
+  interval="timefilter.refreshInterval"
+  on-select="kbnTopNav.toggle('filter')">
```

Comment:
Is there any situation in which we really want to "toggle"? I feel like the intent would be better expressed by using `kbnTopNav.close('filter')`.